components/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,54 @@
+﻿import {apiTryLogin} from "../../api/loginApiClient.module";
+import React, {FormEvent, useState} from "react";
+import scss from "./LoginForm.module.scss";
+
+interface LoginFormProps {
+    userId: string;
+    password: string;
+    setUserId: Function;
+    setPassword: Function;
+}
+
+type FormStatus = "READY" | "FINISHED";
+
+export function LoginForm(props: LoginFormProps): JSX.Element {
+
```

Comment:
You could do some object destructuring here to make it even clearer what LoginForm takes in.
E.g. either 
`LoginForm({userId, password, setUserId, setPassword}: LoginFormPros)...`
or
`const {userId, password, setUserId, setPassword} = props;`
on the first line of the method body.